### ISSUE TITLE
Trust Graph Generation

### ISSUE THREAD

**Issue by massar** [ops-trust/portal#36]
It would be useful to have trust graphs embedded in the site so that one can see how one is connected to another person and thus how one could be introduced to a person one has not directly met.

**Comment by morrowc** [comment]
note that JTK had some code to do this, I think, and make a dot file to
be graphed from?

**Comment by massar** [comment]
Yes, he has. That code renders the full trust-graph, which is something we should have at one point too.

This ticket is more about the direct connections one has and how one can reach somebody whose profile one is peeking at. The actual graph displayed that way will be much smaller.

I'll definitely check JTKs code to see what of his algorithm, can be used for this. Though likely I'll have to exclude various paths that are 'too long' thus it will be quite different ;)

**Comment by morrowc** [comment]
On 01/07/2015 04:04 AM, Jeroen Massar wrote:
> Yes, he has. That code renders the full trust-graph, which is
> something we should have at one point too.
> 
> This ticket is more about the direct connections one has and how one
> can reach somebody whose profile one is peeking at. The actual graph
> displayed that way will be much smaller.
> 
> I'll definitely check JTKs code to see what of his algorithm, can be
> used for this. Though likely I'll have to exclude various paths that
> are 'too long' thus it will be quite different ;)

ah! ok, it seems like you could do some simple sql to pull out the
levels 2 or 3 or N away from you from the vouch table(s).
